jobs/cloud_controller_ng/monit

```diff
@@ -22,7 +22,7 @@
 
 check process cloud_controller_ng
   with pidfile /var/vcap/sys/run/bpm/cloud_controller_ng/cloud_controller_ng.pid
-  start program "/var/vcap/jobs/bpm/bin/bpm start cloud_controller_ng"
+  start program "/var/vcap/jobs/bpm/bin/bpm start cloud_controller_ng" with timeout 60 seconds
```

Comment:
Are you using [BPM](https://github.com/cloudfoundry-incubator/bpm-release) in your environment?